templates/sshd.custom.erb

```diff
@@ -0,0 +1,15 @@
+# This file is being maintained by Puppet.
```

Comment:
Agree that the current tests aren't that useful, esp since the sequence of each section is very important... I'll have a look.